man/systemd.netdev.xml

```diff
@@ -639,6 +639,28 @@
         </listitem>
       </varlistentry>
       <varlistentry>
+        <term><varname>Key=</varname></term>
+        <listitem>
+          <para>The <literal>Key</literal> parameter specifies the same key to use in
+          both directions (InputKey and OutputKey). The <literal>Key</literal> is either a
```

Comment:
always write options with the "=" suffix, and enclose in `<varname>` please. i.e. change here:

`<literal>Key</literal>` → `<varname>Key=</varname>`
`InputKey` → `<varname>InputKey=</varname>`